components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/js/configure-authentication-flow.js

```diff
@@ -645,4 +643,52 @@ function setAttributeStep(element) {
         $(this).attr('checked', false);
     });
     $(element).attr('checked', true);
-}
\ No newline at end of file
+}
+
+var documentBeforeChange;
+
+doc.on("beforeChange", function (document, changeObj) {
+    documentBeforeChange = editorContent;
+    documentBeforeChange = documentBeforeChange.replace(/\n|\t/g, '');
```

Comment:
check if this replace all the characters

Reply:
fixed with e578878 
Now it removes the new lines, tabs first and removes spaces from the string